modules/v2x/common/v2x_proxy_gflags.cc

```diff
@@ -32,16 +32,13 @@ DEFINE_string(grpc_server_port, "50101", "grpc server port");
 DEFINE_string(grpc_debug_server_port, "50102", "grpc server debug port");
 
 // Other Flags
-DEFINE_int64(x2v_trafficlight_timer_frequency, 10,
-             "x2v trafficlight timer frequency");
-DEFINE_bool(debug_flag, false, "debug flag");
-DEFINE_int64(v2x_carstatus_timer_frequency, 10,
-             "v2x carstatus timer frequency");
-DEFINE_string(hdmap_file_name,
-              "/apollo/modules/map/data/sunnyvale_big_loop/base_map.bin",
-              "hdmap file name");
-DEFINE_double(traffic_light_distance, 250.0, "traffic light distance");
-DEFINE_double(heading_difference, 30.0 / 180.0, "max heading difference");
+DEFINE_int64(x2v_traffic_light_timer_frequency, 10,
+             "x2v traffic_light timer frequency");
+DEFINE_int64(v2x_car_status_timer_frequency, 10,
+             "v2x car_status timer frequency");
+DEFINE_double(traffic_light_distance, 250.0, "traffic light distance in m");
+DEFINE_double(heading_difference, 30.0 / 180.0,
+              "max heading difference in rad");
 DEFINE_int64(list_size, 6, "size of list which stores the traffic light data");
 DEFINE_int64(msg_timeout, 250, "timeout value which getting the msg from OBU");
 DEFINE_int64(sim_sending_num, 10, "the max sending times");
```

Comment:
what does `sim` mean here ?

Reply:
Simulation data. After the roadside data is not received, it will simulate the data for a subsequent period of time, so that when the roadside data is unstable, the data on the car is still relatively stable.